tests/e2e/kubectl/resource.go

```diff
@@ -1,11 +1,16 @@
 package kubectl
 
 const (
-	ResourceNode       Resource = "node"
-	ResourceNamespace  Resource = "namespace"
-	ResourcePod        Resource = "pod"
-	ResourceKubevirtVM Resource = "virtualmachines.x.virtualization.deckhouse.io"
-	ResourceVM         Resource = "virtualmachine.virtualization.deckhouse.io"
-	ResourceVMIPClaim  Resource = "virtualmachineipaddressclaims.virtualization.deckhouse.io"
-	ResourceVMIPLeas   Resource = "virtualmachineipaddressleases.virtualization.deckhouse.io"
+	ResourceNode            Resource = "node"
+	ResourceNamespace       Resource = "namespace"
+	ResourcePod             Resource = "pod"
+	ResourceKubevirtVM      Resource = "virtualmachines.x.virtualization.deckhouse.io"
+	ResourceKubevirtVMI     Resource = "virtualmachineinstances.x.virtualization.deckhouse.io"
+	ResourceVM              Resource = "virtualmachine.virtualization.deckhouse.io"
+	ResourceVMIPClaim       Resource = "virtualmachineipaddressclaims.virtualization.deckhouse.io"
+	ResourceVMIPLeas        Resource = "virtualmachineipaddressleases.virtualization.deckhouse.io"
+	ResourceLabelKey        Resource = "os"
+	ResourceLabelValue      Resource = "ubuntu"
+	ResourceAnnotationKey   Resource = "test-annotation"
+	ResourceAnnotationValue Resource = "true"
```

Comment:
These constants are not relevant to this package. And it's not a Resource. Move it to the test package